### DISCUSSION TITLE
connecting with the Spotify web playback SDK

### DISCUSSION THREAD

**Discussion by SagKun** [foobar404/Wave.js#37] [Q&A] [answered]
Hi,

I'm working on a project and I found this awesome library!
Do you think it's possible to connect the visuals with a stream of audio that I get from Spotify?
and If so how do I do that, I tried following the "fromStream" function example from the docs, but I keep getting the next error:
TypeError: Failed to execute 'createMediaStreamSource' on 'AudioContext': parameter 1 is not of type 'MediaStream'.
(working with react btw)

**Comment by foobar404** [comment, marked as answer]
Hey, late response, but you might be able to use the srcObject property on the audio html element. MediaStream is the type returned when you request getUserMedia from the global navigator object. Its used for things like your webcam and microphone, so if its a external stream, it might not work. Need to find a way to pipe the audio into an html audio element.
